src/components/member-card/index.js

```diff
@@ -7,16 +7,32 @@ import PropTypes from 'prop-types';
 import ShowSkills from '@components/member-card/show-skills';
 import SuperUserOptions from '@components/member-card/super-user-options';
 
-const Card = ({ developerInfo, isOptionKey }) => {
+const Card = ({ developerInfo, isOptionKey, colorCombination }) => {
```

Comment:
I am a little confused. We are updating the cards for the users. Why are we adding another prop in `member-card` for it?

Reply:
logic for new-member and new-member was present in the same card to begin with